packages/web/src/pages/profile-page/components/mobile/ProfilePage.tsx

```diff
@@ -222,12 +221,112 @@ const getMessages = ({
 })
 
 const g = withNullGuard((props: ProfilePageProps) => {
-  const { profile, albums, playlists } = props
-  if (profile && albums && playlists) {
-    return { ...props, profile, albums, playlists }
+  const { profile } = props
+  if (profile) {
+    return { ...props, profile }
   }
 })
 
+const PlaylistsTab = ({
+  isOwner,
+  profile,
+  userId
+}: {
+  isOwner: boolean
+  profile: User
+  userId: ID | null
+}) => {
+  const { data: playlists, status: playlistsStatus } = useUserPlaylists({
+    userId: userId ?? null
```

Comment:
don't need the `?? null` just make the hook accept `Id | null | undefined` and rely on the enabled check